File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithSlidesInPresentation/RemoveSlides.php
<?php

namespace Aspose\Slides\WorkingWithSlidesInPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;

class RemoveSlides {

    public static function run($dataDir=null)
    {

        # Using Slides Collection to Remove Slide by Index
        RemoveSlides::remove_slide_by_index($dataDir);

        # Using Slides Collection to Remove Slide by ID
        RemoveSlides::remove_slide_by_id($dataDir);

    }

    public static function remove_slide_by_index($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'Aspose.pptx');

        # Accessing a slide using its slide index
        $slide = $pres->getSlides()->get_Item(0);

        # Removing a slide using its reference
        $pres->getSlides()->remove($slide);

        # Saving the presentation file
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Modified.pptx", $save_format->Pptx);

        print "Removed slide by Index, please check the output file." . PHP_EOL;

    }

    public static function remove_slide_by_id($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'Aspose.pptx');

        # Removing a slide using its slide index
        $pres->getSlides()->removeAt(1);

        # Saving the presentation file
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Modified.pptx", $save_format->Pptx);

        print "Removed slide by ID, please check the output file." . PHP_EOL;


    }

}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithSlidesInPresentation/RemoveSlides.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat
from java.io import FileOutputStream

class RemoveSlides:

    def __init__(self):
                
        # Using Slides Collection to Remove Slide by Index
        self.remove_slide_by_index()

        # Using Slides Collection to Remove Slide by ID
        self.remove_slide_by_id()
        
    def remove_slide_by_index(dataDir):
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/RemoveSlides/'    
        # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'Aspose.pptx')

        # Accessing a slide using its slide index
        slide = pres.getSlides().get_Item(0)

        # Removing a slide using its reference
        pres.getSlides().remove(slide)

        # Saving the presentation file
        save_format = SaveFormat
        pres.save(dataDir + "Modified.pptx", save_format.Pptx)

        print "Removed slide by Index, please check the output file."
        
    def remove_slide_by_id(dataDir):
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/RemoveSlides/'    
        
        # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'Aspose.pptx')

        # Removing a slide using its slide index
        pres.getSlides().removeAt(1)

        # Saving the presentation file
        save_format = SaveFormat
        pres.save(dataDir + "Modified.pptx", save_format.Pptx)

        print "Removed slide by ID, please check the output file."
        



if __name__ == '__main__':        
    RemoveSlides()